src/main/java/flaxbeard/immersivepetroleum/common/util/projector/MultiblockProjection.java

```diff
@@ -186,18 +185,18 @@ private void updateData(){
 		
 		// Align corners first
 		if(!mirrored){
-			switch(rotation){
-				case CLOCKWISE_90:		 this.offset.set(1 - size.getZ(), 0, 0);break;
-				case CLOCKWISE_180:		 this.offset.set(1 - size.getX(), 0, 1 - size.getZ());break;
-				case COUNTERCLOCKWISE_90:this.offset.set(0, 0, 1 - size.getX());break;
-				default:				 this.offset.set(0, 0, 0);break;
+			switch (rotation) {
+				case CLOCKWISE_90 -> this.offset.set(1 - size.getZ(), 0, 0);
+				case CLOCKWISE_180 -> this.offset.set(1 - size.getX(), 0, 1 - size.getZ());
+				case COUNTERCLOCKWISE_90 -> this.offset.set(0, 0, 1 - size.getX());
+				default -> this.offset.set(0, 0, 0);
```

Comment:
Following the formatting or trying to restore kind of the style is not realy your strong suit is it? :D

Reply:
I didn't realize some things were changed - do you have a stylesheet for me to import?

Reply:
I kind of do, but i don't think it's supported by IDEA since im using Eclipse? But even *it* is not *realy* a 1:1 thing to my style. But it's as close to it as i could figure this out Lol